src/transform/FooterReadMore.js

```diff
@@ -0,0 +1,349 @@
+import './FooterReadMore.css'
+
+/**
+ * @typedef {function} SaveButtonClickHandler
+ * @param {!string} title
+ * @return {void}
+ */
+
+/**
+  * @typedef {function} TitlesShownHandler
+  * @param {!string[]} titles
+  * @return {void}
+  */
+
+/**
+   * Display fetched read more pages.
+   * @typedef {function} ShownReadMoreHandler
+   * @param {!Object[]} pages
+   * @param {!Document} document
+   * @param {!string} containerID
+   * @param {SaveButtonClickHandler} saveButtonClickHandler
+   * @param {TitlesShownHandler} titlesShownHandler
+   * @return {void}
+   */
+
+let _saveForLaterString = null
+let _savedForLaterString = null
+const _saveButtonIDPrefix = 'readmore:save:'
+
+/**
+ * Removes parenthetical enclosures from string. 
+ * @param {!string} string
+ * @param {!string} opener
+ * @param {!string} closer
+ * @return {!string}
+ */
+const safelyRemoveEnclosures = (string, opener, closer) => {
```

Comment:
I know you're still working on writing a couple tests. This would probably be more worthwhile to exercise than some of the other methods.